.changeset/sweet-plants-nail.md

```diff
@@ -0,0 +1,23 @@
+---
+"@comet/brevo-admin": minor
+"@comet/brevo-api": minor
+---
+
+Add brevo configuration fields for api key and double opt-in email template.
+Env vars containing this information can be removed and must be removed from the brevo module configuration.
+
+```diff
+- BREVO_API_KEY=apiKey
+- BREVO_DOUBLE_OPT_IN_TEMPLATE_ID=templateId
+```
+
+```diff
+BrevoModule.register({
+    brevo: {
+-       apiKey: config.brevo.apiKey,
+-       doubleOptInTemplateId: config.brevo.doubleOptInTemplateId,
+        //...
+    },
+    //..
+})
+```
```

Comment:
Adding the scope to most resolvers and services is a breaking change. I think there should at least be a major changeset mentioning which this. It doesn't have to be detailed since we know it's not relevant in detail